data/hai missions.txt

```diff
@@ -61,6 +62,7 @@ mission "First Contact: Unfettered"
 	source
 		attributes "unfettered"
 	on offer
+		log "Made contact with a faction of Hai called the Unfettered, who call themselves the true Hai. If they are to be believed, the other Hai were pacified somehow by the Drak long ago."
```

Comment:
Change to `Made contact with a faction of Hai who call themselves the Unfettered, claiming to be the true Hai.`?
I'd remove "somehow" after pacified.